var/spack/repos/builtin/packages/flecsi/package.py

```diff
@@ -69,18 +69,19 @@ class Flecsi(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("boost@1.79.0:", when="@2.2:")
     depends_on("kokkos@3.2.00:", when="+kokkos")
     depends_on("kokkos@3.7:", when="+kokkos @2.3:")
-    depends_on("kokkos +cuda +cuda_constexpr +cuda_lambda", when="+kokkos +cuda")
-    depends_on("kokkos +rocm", when="+kokkos +rocm")
-    depends_on("kokkos +openmp", when="+kokkos +openmp")
+    depends_on("kokkos@3.7:", when="@2.3.2:")
```

Comment:
Isn't the unconditional Kokkos dependency a `2.4` thing?

Reply:
Yes, but since there is neither a `2.4` or a `2.3.2`, I'm going with the next smallest increment.